src/util/psbt/mod.rs

```diff
@@ -405,6 +410,7 @@ mod tests {
             }],
         };
         let encoded = ::serde_json::to_string(&psbt).unwrap();
+        println!("encoded PSBT: {}", encoded);
```

Comment:
debug printf

Reply:
Is that a problem? It's in a unit test, so it will only be printed when used with `--nocapture`. I can remove it if it's a problem. When changing any of the PSBT fields, this helps looking at how the JSON actually looks.

Reply:
(F.e. the `Transactioin.witness` field still is `Vec<Vec<u8>>` which doesn't print as hex. I tried fixing that too, but I got tired of fighting serde for a while, might do that in another PR.)

Reply:
Yeah, that's fair, you can leave it in.